src/app/actions/NewsActions.js

```diff
@@ -1,11 +1,11 @@
 import Request from 'superagent';
 import dispatcher from '../dispatcher';
 
-export const displayNews = (id, sort, name) => {
-  const url = 'https://newsapi.org/v1/articles?apiKey=213327409d384371851777e7c7f78dfe';
+export const displayNews = (id, sortArr, name) => {
+  const url = `https://newsapi.org/v1/articles?apiKey=${process.env.NEWS_API_KEY}`;
   Request.get(url)
     .query({ source: id })
-    .query({ sortBy: sort })
+    .query({ sortBy: sortArr })
```

Comment:
Nice one, but I believe you can pass your queries as a single query object. You might want to try that option.